dev/com.ibm.ws.concurrent_fat_ejb/fat/src/com/ibm/ws/concurrent/ejb/fat/ConcurrentEJBTest.java

```diff
@@ -38,12 +42,23 @@
 
     public static final String APP_NAME = "concurrent";
 
+    @ClassRule
+    public static RepeatTests r = RepeatTests
+                    .withoutModification()
+                    .andWith(new JakartaEE9Action());
+
     @Server("com.ibm.ws.concurrent.fat.ejb")
     @TestServlet(servlet = ConcurrentFATServlet.class, path = APP_NAME)
     public static LibertyServer server;
 
     @BeforeClass
     public static void setUp() throws Exception {
+        ServerConfiguration config = server.getServerConfiguration();
+        if (config.getFeatureManager().getFeatures().contains("servlet-5.0") && config.getFeatureManager().getFeatures().remove("ejblite-3.2")) {
```

Comment:
Eventually this check will evaluate to `false` once `ejblite-4.0` is added to `JakartaEEE9Action`, so instead lets use this more direct check:
```java
if (JakartaEE9Action.isActive()) {
  // ...
}
```

Reply:
It turns out the whole issue the test was trying to work around here (jndi-1.0 present in ejblite-3.2 but absent from ejblite-4.0) is no longer the case, so this code back can be deleted entirely.  I've added a commit for that.